Can you check travis
```
flake8 --max-line-length=120 $(find . \( -name 'pre-*.py' -or -name 'post-*.py' \) -exec grep -q openupgrade {} \; -print)
./addons/purchase_requisition/migrations/10.0.0.1/post-migration.py:14:27: E261 at least two spaces before inline comment
The command "flake8 --max-line-length=120 $(find . \( -name 'pre-*.py' -or -name 'post-*.py' \) -exec grep -q openupgrade {} \; -print)" exited with 1.
```
 and, as an additional fix, can you set 'company_id' to False in the created product? It should work better in multicompany context.

--- a/addons/purchase_requisition/migrations/10.0.0.1/post-migration.py
+++ b/addons/purchase_requisition/migrations/10.0.0.1/post-migration.py
@@ -8,16 +8,24 @@ from openupgradelib import openupgrade
 
 def fill_product(env):
     """Fill with an pre-created product as it's required."""
-    product = env['product.product'].create({
-        'active': False,
-        'name': 'OU purchase requisition line wildcard',
-    })
-    openupgrade.logged_query(
-        env.cr, """
-        UPDATE purchase_requisition_line
-        SET product_id = %s
-        WHERE product_id IS NULL;""", (product.id, ),
-    )
+    if env['purchase.requisition.line'].search_count(
+            [('product_id', '=', False)]) > 0:
+        categ_id = env["product.template"]._get_default_category_id()
+        if not categ_id:
+            categ_id = env["product.category"].search(
+                [], limit=1, order='id asc').id
+        product = env['product.product'].create({
+            'active': False,
+            'categ_id': categ_id,  # categ_id is required
+            'company_id': False,  # tacke into account multi-company context
+            'name': 'OU purchase requisition line wildcard',
+        })
+        openupgrade.logged_query(
+            env.cr, """
+            UPDATE purchase_requisition_line
+            SET product_id = %s
+            WHERE product_id IS NULL;""", (product.id, ),
+        )
 
 
 @openupgrade.migrate()

